src/components/Transcript/Transcript.md

```diff
@@ -54,25 +54,25 @@ import Transcript from '../Transcript/Transcript';
           url: 'https://dlib.indiana.edu/iiif_av/iiif-player-samples/transcripts/transcript_ms.docx',
         },
         {
-          // Directly feeding a plain text file from a server
-          title: 'Transcript in Plain Text',
-          url: 'https://dlib.indiana.edu/iiif_av/iiif-player-samples/transcripts/transcript_plain.txt',
-        },
-        {
-          // External plain text transcript fed through `rendering` prop in a IIIF manifest
+          // External plain text transcript fed through `annotations` prop in a IIIF manifest
           title: 'External text transcript',
-          url: 'https://dlib.indiana.edu/iiif_av/iiif-player-samples/transcripts/transcript-manifest-rendering.json', // URL of the manifest
+          url: 'https://dlib.indiana.edu/iiif_av/iiif-player-samples/transcripts/transcript-manifest.json', // URL of the manifest
```

Comment:
I see that the json fixture filename is renamed here but it looks like the actual file needs to be renamed to match this change.  Same for `transcript-canvas-rendering.json` -> `transcript-canvas.json` below.

Reply:
Thanks for the review @cjcolvar.
Do you mean the file in the `test_data` directory in the code?

Reply:
Sorry, yes.  (I think I had changed this comment to be clearer but it got reverted. :slightly_frowning_face: )  
I was assuming the filename in this url would match `src/json/test_data/transcript-manifest-rendering.js` (except for the js/json difference).  Is that correct or am I linking things that shouldn't be?

Reply:
No these urls do not match within the /src/json directory.  I cleaned up that data and put these on maple. The files in the /src/json are for testing purposes. I'll change them, since it's good idea to have them renamed.